File: CorrespondenceSystem/CorrespondenceSystem.Web/Modules/ServerTypes/SignDB/SignColumns.ts
import { ColumnsBase, fieldsProxy } from "@serenity-is/corelib";
import { Column } from "@serenity-is/sleekgrid";
import { SignRow } from "./SignRow";

export interface SignColumns {
    Title: Column<SignRow>;
    SignAttachment: Column<SignRow>;
    IsLast: Column<SignRow>;
}

export class SignColumns extends ColumnsBase<SignRow> {
    static readonly columnsKey = 'SignDB.Sign';
    static readonly Fields = fieldsProxy<SignColumns>();
}

File: CorrespondenceSystem/CorrespondenceSystem.Web/Modules/SignDB/Sign/SignColumns.cs
using Serenity.ComponentModel;
using System;
using System.ComponentModel;

namespace CorrespondenceSystem.SignDB.Columns;

[ColumnsScript("SignDB.Sign")]
[BasedOnRow(typeof(SignRow), CheckNames = true)]
public class SignColumns
{
    [EditLink]
    public string Title { get; set; }
    public string SignAttachment { get; set; }
    //public DateTime CreatedDate { get; set; }
    public bool IsLast { get; set; }
}